tests/wpt/metadata-css/cssom-1_dev/html/interfaces.htm.ini

```diff
@@ -204,3 +204,9 @@
   [PseudoElement interface: attribute usedStyle]
     expected: FAIL
 
+  [CSSMediaRule interface: existence and properties of interface object]
```

Comment:
To be clear, my issue isn't autosyncing -- I've had to do that before and I just make PRs upstream or to Gecko.

My issue is that this is a cssom-1 test, and cssom-1 defines CSSMediaRule as inheriting from CSSGroupingRule, but css-conditional-3 supersedes that, but that spec is CR and that specific part hasn't been implemented in many browsers yet.

I'm not sure what specs the tests track, especially when they're in one folder but affected by newer (but more unstable) specs.